src/server/repository/public/user/count.ts

```diff
@@ -41,12 +45,28 @@ export const count = async (props: Props, client: BaseProtocol = DB): Promise<Re
               `)} ${groupByRole ? `group by ur.role` : ''}`
   }
 
-  const queryTotals = getQuery()
   const queryRoles = `with counts as (${getQuery(true)})
-                      select jsonb_object_agg(counts.role, counts.totals) as result
-                      from counts`
+  select jsonb_object_agg(counts.role, counts.totals) as result
+  from counts`
+
+  const { query: subQueryTotals, queryParams: queryTotalsParams } = buildGetManyQuery({
+    administrators,
+    assessment,
+    countries,
+    cycle,
+    fullName,
+    roles,
+    statuses,
+  })
+
+  const queryTotals = `
+  select count(*) as total
+  from (
+      ${subQueryTotals}
+  ) as users;
+  `
```

Comment:
The query that calculates the `total` must be exactly the same as the one that gets the data (except for the limit and offset), otherwise any incongruences can result in the table paginator not working correctly. e.g it showing that there are 5 pages but in reality there are only 3. This happened to me, so I introduced this change.